### PR TITLE
Dm testing

### PR DESCRIPTION
**Pull request by diyanamendoza**
## Description

Per office hour convo on 1/27, this PR provides a test for the cleanData function and removes trim from the function itself due to regex coverage for removing spaces globally.

## Related Issue

cleanData util was written for #22 

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|   | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|  ✓  | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

## Testing Steps / QA Criteria

Pull down branch and run npm test in your terminal - tests for the cleanData.test.js file should pass.
 
Screen Shot 2022-01-27 at 11 26 07 AM

### CODE REVIEW

File: src/utils/cleanData.test.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import cleanData from './cleanData';
```
Comment (chriskorsak) [inline src/utils/cleanData.test.js:1]: Very cool! This is a great contribution to the project.

**Review by chriskorsak — APPROVED** [review]
Great job on creating a test for the cleanData function! Works great. I altered a .toBe() argument on one of the tests just to see what happens, and there we go, error!

**Review by k-monnik — APPROVED** [review]
This is cool! Thanks for creating!

File: package.json [hunk 2]
```diff
@@ -15,6 +15,7 @@
     "@testing-library/react": "^11.2.2",
     "@testing-library/user-event": "^12.6.0",
     "husky": "^4.3.6",
+    "jest": "^26.6.0",
```
Comment (aprilleone) [inline package.json:18]: Wooohooo! Jest!

File: src/utils/cleanData.test.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  expect(cleanData('B&$(#&^EaNs))))')).toBe('beans');
+});
+
+test('SWEET  potato returns sweetpotato', () => {
+  expect(cleanData('SWEET  potato')).toBe('sweetpotato');
+});
+
+test('tofu  returns tofu', () => {
+  expect(cleanData(' tofu ')).toBe('tofu');
+});
```
Comment (aprilleone) [inline src/utils/cleanData.test.js:21]: Awesome! This test is such a great clean way to verify what your expectations are into reality and the rules stay applied through the life of the app. Nicely done.

Going forward, we can identify what other functions could be tested?

**Review by aprilleone — APPROVED** [review]
We have tests!

**Pull request merged**